### PR TITLE
Added Three Liner examples

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3526

#### Changes proposed in this Pull Request:
- Added examples for Three Liner

#### Screenshots:
Screenshot 2022-09-27 at 2 01 57 PM

#### To Test:
- yarn start:reactdev

### CODE REVIEW

**Comment by ektasawant** [comment]
Screenshot 2022-09-27 at 2 57 34 PM
Formatting is breaking here.

**Comment by bkarambe** [comment]
> Screenshot 2022-09-27 at 2 57 34 PM
> 
> Formatting is breaking here.

Updated

**Review by ektasawant — DISMISSED** [review]

**Comment by huayunh** [comment]
@ektasawant maybe you were mentioning the issue logged in #485 ?

**Comment by huayunh** [comment]
Typo
image

File: docs/src/componentDocs/ThreeLiner/markdown/ThreeLinerExamples.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { BasicThreeLiner } from '../examples/BasicThreeLiner';
+import { ThreeLinerWithCustomContent } from '../examples/ThreeLinerWithCustomContent';
+
+# Basic Three Liner
+
+The `<ThreeLiner>` is commonly used within a [`<AppBar>`](/components/app-bar/api-docs). It accepts a `title`, `subtitle` and `info` props.
```
Comment (huayunh) [inline docs/src/componentDocs/ThreeLiner/markdown/ThreeLinerExamples.mdx:6]:
```suggestion
The `<ThreeLiner>` is commonly used within a [`<AppBar>`](/components/app-bar/api-docs). It accepts `title`, `subtitle`, and `info` as props.
```

**Comment by huayunh** [comment]
Everything else works fine.

File: docs/src/componentDocs/ThreeLiner/examples/ThreeLinerWithCustomContent.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { Box } from '@mui/material';
+import { CodeBlock, CodeBlockActionButtonRow } from '../../../shared';
+import * as colors from '@brightlayer-ui/colors';
+import { ThreeLinerWithCustomContentExample } from './ThreeLinerWithCustomContentExample';
+
+const codeSnippet = `<ThreeLiner
+    title="title"
+    subtitle="subtitle"
+    info={
+        <ToolbarMenu
```
Comment (ektasawant) [inline docs/src/componentDocs/ThreeLiner/examples/ThreeLinerWithCustomContent.tsx:11]:
```suggestion
       <ToolbarMenu
            label="info"
            menuGroups={[
                {
                    items: [
                        {
                            title: 'Menu Item 1'
                        },
                        {
                            title: 'Menu Item 2'
                        },
                        {
                            title: 'Menu Item 3'
                        },
                    ],
                },
            ]}
        />
```

Comment (ektasawant) [reply docs/src/componentDocs/ThreeLiner/examples/ThreeLinerWithCustomContent.tsx:11]: @bkarambe Higlighling is breaking. You need to update data-line attr.
Screenshot 2022-09-28 at 2 12 33 PM

**Review by ektasawant — APPROVED** [review]

**Pull request merged**